charts/rancher-logging/0.1.2/charts/fluentd/charts/fluentd-linux/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+name: fluentd-linux
+version: 0.0.1
+appVersion: 1.1.3
```

Comment:
These versions don't line up with the versions mentioned in the directory structure

Reply:
Hi @drpebcak we have a version for rancher-logging charts, which is under `rancher/system-charts/charts/rancher-logging/0.1.2/Chart.yaml`, for sub-charts fluentd/fluentd-tester/log-aggregator, if the charts under their directory changed, then we change the version. For sub-charts like `rancher-logging/0.1.2/charts/fluentd/charts/fluentd-linux` if itself changed, then we change version. The appVersion is bind to the fluentd version, so I change it to `1.6.3` and update some charts version to 0.0.2. Is this ok?

Reply:
Makes sense. I see now that 0.1.2 is not a released version of the `rancher-logging` chart